addons/payment_mollie_official/data/method_data.xml

```diff
@@ -0,0 +1,180 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
+        <record id="payment_method_ideal" model="payment.icon">
+            <field name="name">iDEAL</field>

```

Comment:
there's quite a few duplicates compared to payment/data/payment_icon_data.xml

Reply:
I've removed the duplicates, don't be a drama queen though, it where just two haha ;)